model:
  class_filename: "/home/developer/object_detection/catkin_ws/src/object_detection/model/classification_classes_ILSVRC2012.txt"
  model_filename: "/home/developer/object_detection/catkin_ws/src/object_detection/model/DenseNet_121.prototxt"
  config_file: "/home/developer/object_detection/catkin_ws/src/object_detection/model/DenseNet_121.caffemodel"
  score_threshold: 0.4
display:
  imshow: true

# cv-camera params
cv_camera:
  rate: 30
  device_id: 0
  frame_id: "camera"
  image_width: 640
  image_height: 480
